scripts/android-nightly.patch

```diff
@@ -1,15 +1,15 @@
 diff --git a/example/android/gradle/wrapper/gradle-wrapper.properties b/example/android/gradle/wrapper/gradle-wrapper.properties
-index 4e86b92..37aef8d 100644
+index 66c01cf..3fa8f86 100644
 --- a/example/android/gradle/wrapper/gradle-wrapper.properties
 +++ b/example/android/gradle/wrapper/gradle-wrapper.properties
 @@ -1,6 +1,6 @@
  distributionBase=GRADLE_USER_HOME
  distributionPath=wrapper/dists
 -distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.2-bin.zip
-+distributionUrl=https\://services.gradle.org/distributions/gradle-8.1.1-bin.zip
++distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

Comment:
nit: just flagging, making sure you want to keep the 8.4 around (might be GH messing things up)

Reply:
This is intentional since we want to sync to upstream for nightly builds: https://github.com/facebook/react-native/blob/60f5a80c1d5900e5213aabd3d3a762174996cfd1/packages/react-native/template/android/gradle/wrapper/gradle-wrapper.properties#L3